pkgs/development/libraries/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs, lib, stdenv, fetchFromGitLab }:
+stdenv.mkDerivation rec {
+  pname = "wireplumber";
+  version = "0.4.5";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "pipewire";
+    repo = "wireplumber";
+    rev = version;
+    sha256 = "sha256-EQw1hVQ0rsB9CWOBtcP1enpIwkHmNTr3AJm3rFiJpsw=";
+  };
+
+  mesonFlags = [
+    "-Dsystem-lua=true"
+    "-Delogind=disabled"
+  ];
+
+  nativeBuildInputs = with pkgs; [
+    meson
+    pkg-config
+    ninja
+
+    # for docs
+    (python39.withPackages (ps: with ps; [
```

Comment:
imo it'd be nice if this were conditional on an `enableDocs` arg

Reply:
Fixed this too.